docs: fix mistakes and typos; improve readability with the help of AI tools (#50)
goravel/docs: README.md

## About Goravel

> Goravel is a web application framework with complete functions and excellent scalability. It serves as an ideal kick-start for Gophper to rapidly build their own web applications.

Reviewer: ```suggestion
Goravel is a web application framework with complete functions and excellent scalability. As a starting scaffolding to help Gopher quickly build their own applications.
```

> The framework's design is consistent with [Laravel](https://github.com/laravel/laravel), simplifying the learning curve for PHPers. Moreover, it encourages experimentation with Golang. Kudos to Laravel!

Reviewer: ```suggestion
The framework's design is consistent with [Laravel](https://github.com/laravel/laravel), simplifying the learning curve for PHPers. Kudos to Laravel!
```
